xwiki-platform-core/xwiki-platform-user/xwiki-platform-user-profile/xwiki-platform-user-profile-ui/src/main/resources/XWiki/XWikiUserSheet.xml

```diff
@@ -448,17 +448,23 @@ return XWiki;
   width: 13em;
 }
 
-#avatar p {
-  text-align: center;
-}
+#avatar {
+  p {
+    text-align: center;
+  }
```

Comment:
Rulesets that are next to each other should be separated with an empty line. Here, this would mean adding back a line between the `p` ruleset and the `img` one :)

> Separate adjacent rulesets with an empty line.

From https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/XhtmlCssCodeStyle/